clients/java/src/main/java/io/camunda/zeebe/client/api/search/response/DecisionRequirements.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.search.response;
+
+public interface DecisionRequirements {
```

Comment:
I think it would be better that this interface extends `io.camunda.zeebe.client.api.response.DecisionRequirements` and adds the missing methods returned by the search here (for now there is none)
Same thing was done with `ProcessInstance`

Reply:
Thanks for the suggestion 💯  applied on https://github.com/camunda/camunda/pull/20786/commits/7c2e1f281e1a07162a8b8c0f69b24ed2d84fd069